VulnerableReader

```diff
@@ -0,0 +1,14 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+
+public class VulnerableReader {
+    public static void main(String[] args) throws Exception {
+        BufferedReader reader = new BufferedReader(new FileReader("secret.txt"));
+        String line;
+        while ((line = reader.readLine()) != null) {
+            System.out.println(line);
+        }
+        // This line will not be executed because the program terminates at this point
+        try (BufferedReader ignored = new BufferedReader(new FileReader("large_file.txt"))) {}
+    }
+}
```

Comment:
```suggestion
import java.io.BufferedReader;
import java.io.FileReader;

public class VulnerableReader {
    public static void main(String[] args) throws Exception {
        BufferedReader reader = new BufferedReader(new FileReader("secret.txt"));
        String line;
        while ((line = reader.readLine()) != null) {
            System.out.println(line);
        }
        // This line will not be executed because the program terminates at this point
        try (BufferedReader ignored = new BufferedReader(new FileReader("large_file.txt"))) {}
    }
}```